src/components/sbb-autocomplete/sbb-autocomplete.e2e.ts

```diff
@@ -1,190 +1,183 @@
     await waitForCondition(() => didCloseEventSpy.events.length === 2);
-    expect(didCloseEventSpy).toHaveReceivedEventTimes(2);
-    expect(input.getAttribute('aria-expanded')).toEqual('false');
+    expect(didCloseEventSpy.count).to.be.equal(2);
+    expect(input).to.have.attribute('aria-expanded', 'false');
 
-    await input.click();
-    await page.waitForChanges();
+    input.click();
     await waitForCondition(() => willOpenEventSpy.events.length === 3);
-    expect(willOpenEventSpy).toHaveReceivedEventTimes(3);
-    await page.waitForChanges();
+    expect(willOpenEventSpy.count).to.be.equal(3);
     await waitForCondition(() => didOpenEventSpy.events.length === 3);
-    expect(didOpenEventSpy).toHaveReceivedEventTimes(3);
-    expect(input.getAttribute('aria-expanded')).toEqual('true');
+    expect(didOpenEventSpy.count).to.be.equal(3);
+    expect(input).to.have.attribute('aria-expanded', 'true');
+
+    // Simulate backdrop click
+    sendMouse({ type: 'move', position: [350, 25] });
+    sendMouse({ type: 'down' });
+    sendMouse({ type: 'up' });
 
-    const button = await page.find('button');
-    await button.click();
     await waitForCondition(() => willCloseEventSpy.events.length === 3);
-    expect(willCloseEventSpy).toHaveReceivedEventTimes(3);
-    await page.waitForChanges();
+    expect(willCloseEventSpy.count).to.be.equal(3);
     await waitForCondition(() => didCloseEventSpy.events.length === 3);
-    expect(didCloseEventSpy).toHaveReceivedEventTimes(3);
-    expect(input.getAttribute('aria-expanded')).toEqual('false');
+    expect(didCloseEventSpy.count).to.be.equal(3);
+    expect(input).to.have.attribute('aria-expanded', 'false');
   });
 
   it('select by mouse', async () => {
-    const willOpenEventSpy = await page.spyOnEvent(events.willOpen);
-    const didOpenEventSpy = await page.spyOnEvent(events.didOpen);
-    const optionSelectedEventSpy = await page.spyOnEvent(optionEvents.optionSelected);
+    const willOpenEventSpy = new EventSpy(events.willOpen);
+    const didOpenEventSpy = new EventSpy(events.didOpen);
+    const optionSelectedEventSpy = new EventSpy(optionEvents.optionSelected);
 
-    await input.focus();
-    await page.waitForChanges();
+    input.focus();
     await waitForCondition(() => willOpenEventSpy.events.length === 1);
-    expect(willOpenEventSpy).toHaveReceivedEventTimes(1);
-    await page.waitForChanges();
+    expect(willOpenEventSpy.count).to.be.equal(1);
     await waitForCondition(() => didOpenEventSpy.events.length === 1);
-    expect(didOpenEventSpy).toHaveReceivedEventTimes(1);
+    expect(didOpenEventSpy.count).to.be.equal(1);
 
-    await element.press('ArrowDown');
-    await element.press('ArrowDown');
-    await page.waitForChanges();
-    await element.press('Enter');
-    await page.waitForChanges();
+    await sendKeys({ press: 'ArrowDown' });
+    await sendKeys({ press: 'ArrowDown' });
+    await element.updateComplete;
```

Comment:
Not sure here. In theory, the 'ArrowDown' changes the state of the component that should be rendered before new inputs are received.
In this specific case, since the keys only affect the 'logic' and are not dependent on any rendered element, we are fine even if we remove the extra updateComplete (which I'll do).